liquibase-core/src/main/java/liquibase/changelog/ChangelogRewriter.java

```diff
@@ -15,6 +15,12 @@
 import java.util.regex.Pattern;
 
 public class ChangelogRewriter {
+
+    public static final String XSD_PATTERN_STRING = "([dbchangelog|liquibase-pro])-3.[0-9]?[0-9]?.xsd";
```

Comment:
I was thinking maybe can append a REGX/REGEX at the end of these regex constants. Also, I would lightly change the name of some of the patterns changing `XSD_PATTERN` for `XSD_NAME_PATTERN`. What do you think about it?